bitnami/ejbca/requirements.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: mariadb
+  repository: https://charts.bitnami.com/bitnami
+  version: 7.5.2
```

Comment:
[The latest version is 7.6.1](https://github.com/bitnami/charts/commit/090a91f64e2b8c4c77aa3a07f0b672367d34f42e), maybe you want to run `helm dep update` again

Reply:
Done. Up and running with the latest versions.